elated-egrets/bangazon-internal-site
# Rm training program view

## DESCRIPTION
Added views for training program list and add w/ tests

## How to pull down
Commands

```
git fetch --all
git checkout rm-training-program-view
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. go to /training_programs and see if there is a list of training programs
1. go to /training_programs/add and see if the form will add a new trining program
1. from the root directory run `python manage.py test` and see if the tests pass

## What you changed
added form for training program as well as list view, and tests for the model and views.

## What was added to read me
information about url formatting

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
